src/xcode/ENA/ENA/Source/HTTPClient/Service/ServiceError.swift

```diff
@@ -8,6 +8,16 @@ import ENASecurity
 The errors that can occur while using the service and calling http methods.
 */
 enum ServiceError<RE>: Error, Equatable where RE: Error {
+	// we need this property for nested errors
+	var errorDescription: String {
```

Comment:
```suggestion
enum ServiceError<RE>: Error, CustomStringConvertible, Equatable where RE: Error {

		// MARK: - Protocol CustomStringConvertible

		var description: String {
```

I think it would be more fitting to use the `CustomStringConvertible` protocol as it is not really a full errorDescription but rather a nicer identifier.

And it would be nice to move the implementation below the cases, next to the `Equatable` implementation.